authors/apps/articles/urls.py

```diff
@@ -1,21 +1,30 @@
 from django.urls import path
+
 from .views import (
     ArticlesListCreateAPIView,
     ArticleRetrieveUpdateDestroy,
     ArticleRetrieveBySlugAPIView,
+    CommentListCreateView,
+    ThreadListCreateView,
+    CommentRetrieveDeleteView
 )
 
 urlpatterns = [
     # GET/POST api/articles
-    path('', ArticlesListCreateAPIView.as_view(), name='list_create'),
-
+    path('articles', ArticlesListCreateAPIView.as_view(), name='list_create'),
     # GET api/articles/id
-    # PUT api/articles/id
-    # DELETE api/articles/id
-    path('/<int:pk>', ArticleRetrieveUpdateDestroy.as_view(),
+    path('articles/<int:pk>', ArticleRetrieveUpdateDestroy.as_view(),
          name='article_by_id'),
-
     # GET api/articles/slug
-    path('/<slug:slug>', ArticleRetrieveBySlugAPIView.as_view(),
+    path('articles/<slug:slug>', ArticleRetrieveBySlugAPIView.as_view(),
```

Comment:
Why are you adding this?

Reply:
It is how I had initially written the path before rebasing.